storage/rocksdb/ha_rocksdb.cc

```diff
@@ -403,7 +403,7 @@ static uint64_t rocksdb_info_log_level;
 static char *rocksdb_wal_dir;
 static char *rocksdb_persistent_cache_path;
 static uint64_t rocksdb_index_type;
-static char rocksdb_background_sync;
+static uint32_t rocksdb_flush_log_at_trx_commit;
```

Comment:
I'd prefer types aligned with MYSQL_*VAR macros here, but looks like upstream has chosen otherwise, no point in changing